### PR TITLE
Feature/home screen

### PR DESCRIPTION
**Pull request by nachosoloaga**
### Links (Jira/Trello ticket and other relevant links):
Trello: https://trello.com/b/77bHIxus/rootstrap-target
Figma: https://www.figma.com/file/Hlq710GJ4jCPU9EDgQS1Dx/Target-Web?node-id=0%3A2189

### What & Why:
This PR involves the addition of a welcome screen when a given user is logged in. A map should be seen, along with initial tips and a confirm button, so the user can easily identify multiple targets in the future.

## Tasks:
- [x] Tested in different resolutions
- [ ] Tested with SSR
- [x] Tested on Safari browser
- [x] Design reviewed with design team

## Notes:

- Next PRs will involve changes in the Router component, in order to change to a more declarative way of defining routes, and a GuestLayout and PrivateLayout component, so duplicate code between pages can be extracted.

- On a previous PR a menu button was added so the user can logout, and navigate to 'About' and 'Contact' sections.

## Screenshots:

image

### CODE REVIEW

File: src/components/user/LogoutButton.js [hunk 1]
```diff
@@ -8,9 +8,9 @@ const LogoutButton = () => {
   const logoutRequest = useDispatch(logout);
 
   return (
-    <button onClick={logoutRequest} type="button">
+    <a style={{ cursor: 'pointer' }} onClick={logoutRequest}>
```
Comment (matteo95g) [inline src/components/user/LogoutButton.js]: I think it's better to create a class named `cursor-pointer` or something like that and use it here

File: src/locales/es.js [hunk 2]
```diff
@@ -27,6 +27,8 @@ export default {
   'app.legend': 'Encuentra gente cerca y conecta',
   'app.info': `Crea un objetivo en cualquier parte del mapa, indica tu interés: Travel, Dating, Music, etc
   y empieza a conectar con personas que lo comparten.`,
+  'app.welcome.item1': `Crea un objetivo haciendo click en cualquier parte del mapa, indica el área y una temática: Travel, Dating, Music, etc.`,
```
Comment (JulianPasquale) [inline src/locales/es.js]: Can we use single quotes here? I'd use that only when using interpolation

```suggestion
  'app.welcome.item1': 'Crea un objetivo haciendo click en cualquier parte del mapa, indica el área y una temática: Travel, Dating, Music, etc.',
```

File: src/pages/HomePage/HomeInfo.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      <div className="home-info-text-container">
+        <Smilies />
+        <div className="info-text">
+          <h1>{APP_TITLE.toUpperCase()}</h1>
+          <div className="welcome-text">
+            <h2>
+              <FormattedMessage id="app.legend" />
+            </h2>
+            <p>
+              <img style={{ marginRight: '0.5rem' }} src={Point} alt="Point" />
```
Comment (JulianPasquale) [inline src/pages/HomePage/HomeInfo.js]: Can we create a class for this styles?

File: src/components/common/Map/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { MapContainer, TileLayer } from 'react-leaflet';
+
+const Map = () => {
+  return (
+    <MapContainer
+      style={{ height: '100vh', width: '65%' }}
```
Comment (JulianPasquale) [inline src/components/common/Map/index.js:7]: Can we create a class for this styles?

**Review by JulianPasquale — APPROVED** [review]

**Review by matteo95g — APPROVED** [review]

**Pull request merged**